install_functions/installMain.sh

```diff
@@ -236,6 +262,9 @@ function main() {
 
     common_checkInstalled
     checks_arguments
+    if [ -n "${development}" ]; then
+        checks_filebeatURL
+    fi
```

Comment:
Checking the Filebeat URL is done for all cases, not just for development, I think.

Reply:
It is not necessary to check it in production case. Notice that:
`source_branch="v${wazuh_version}"`, then
`filebeat_wazuh_template="https://raw.githubusercontent.com/wazuh/wazuh/${source_branch}/extensions/elasticsearch/7.x/wazuh-template.json"
`
The template will be downloaded from the tag, then, it is not necessary to change the filebeat template URL